app/views/favorites/index.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="section-box text-center small-10 large-4 large-centered small-centered columns">
+  <h1 class="section-title">Favorite Tattoos</h1>
+</div>
+
+<div class="row">
+  <div class="large-12 columns">
+      <div class="row">
+        <div class="large-12 columns">
+          <div class="row">
```

Comment:
I'd say you have too many nested divs here that aren't accomplishing much! I'd say get rid of lines 8 and 9